src/infra/marketplace/marketplaceQueryRepo_test.go

```diff
@@ -12,14 +15,30 @@ func TestVirtualHostQueryRepo(t *testing.T) {
 	marketplaceQueryRepo := NewMarketplaceQueryRepo(persistentDbSvc)
 	testHelpers.LoadEnvVars()
 
-	t.Run("ReadCatalogItems", func(t *testing.T) {
-		catalogItems, err := marketplaceQueryRepo.ReadCatalogItems()
+	t.Run("Read", func(t *testing.T) {
```

Comment:
You added new methods but your unit test only tests the first one. Why?

Reply:
I added the new methods to the test.